src/logger/index.js

```diff
@@ -2,19 +2,21 @@ const winston = require('winston');
 
 const SPLAT = Symbol.for('splat');
 
-let logLevel;
+let logLevel = process.env.LOG_LEVEL;
```

Comment:
logLevel should have a default defined here and internally that value must be validated to match levels

Reply:
If someone uses this system variable, he will know what to enter there. In every other case the default handling is used.
This is not a finale variant it needs to be reworked, later.